src/main/java/org/kohsuke/github/GHPerson.java

```diff
@@ -52,7 +52,7 @@ protected synchronized void populate() throws IOException {
      */
     public synchronized Map<String,GHRepository> getRepositories() throws IOException {
         Map<String,GHRepository> repositories = new TreeMap<String, GHRepository>();
-        for (GHRepository r : listRepositories()) {
+        for (GHRepository r : listRepositories(100)) {
```

Comment:
That would be for `listRepositories(int pageSize)`:

> Unlike `getRepositories()`, this does not wait until all the repositories are returned.

So what do you think needs to be amended? `listRepositories` overloads produce incremental results, whereas `getRepositories` waits for the full list. I am not changing those semantics (that I know of!); I am just asking for larger batches, on the grounds that an organization with 300 repositories is better enumerated using 3 requests returning 100 entries each, rather than 10 requests returning 30 entries each.